apps/fluent-tester/src/FluentTester/theme/applyBrand.ts

```diff
@@ -64,7 +64,7 @@ const brandColors: BrandRamps = {
 
 export const brandOptions = Object.keys(brandColors).map((brand) => ({ label: brand, value: brand }));
 
-export const applyBrand = (currentBrand: string) => {
+export const applyBrand = (currentBrand: string): { [key: string]: any } => {
```

Comment:
nit: this could be PartialTheme

Reply:
...I swear that caused a build break w/ another type. correcting